build.gradle

```diff
@@ -108,14 +108,15 @@ dependencies {
     implementation group: 'org.tinylog', name: 'tinylog-impl', version: tinyLogVersion
 
     // JUnit test framework.
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.11.4'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.12.0'
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
```

Comment:
Without this, [tests tasks were failing](https://github.com/TrueLayer/truelayer-java/actions/runs/13702432356/job/38319392390) with new Junit version